src/components/layers/basemaps/BasemapCard.js

```diff
@@ -11,71 +12,65 @@ import {
     selectBasemap,
 } from '../../../actions/basemap';
 import { VECTOR_STYLE } from '../../../constants/layers';
+import useBasemapConfig from '../../../hooks/useBasemapConfig';
 
-// Basemap card shown in left layers panel
 const BasemapCard = props => {
     const {
-        name,
         subtitle = i18n.t('Basemap'),
-        opacity,
-        isExpanded,
-        isVisible,
         toggleBasemapExpand,
         toggleBasemapVisibility,
         changeBasemapOpacity,
+        selectBasemap,
     } = props;
-
-    const hasOpacity = props.config.type === VECTOR_STYLE ? false : true;
+    const basemap = useBasemapConfig(props.basemap);
 
     return (
-        <LayerCard
-            hasOpacity={hasOpacity}
-            title={name}
-            subtitle={subtitle}
-            opacity={opacity}
-            isExpanded={isExpanded}
-            isVisible={isVisible}
-            onOpacityChange={changeBasemapOpacity}
-            toggleExpand={toggleBasemapExpand}
-            toggleLayerVisibility={toggleBasemapVisibility}
-        >
-            <BasemapList
-                selectedID={props.basemap.id}
-                basemaps={props.basemaps}
-                selectBasemap={props.selectBasemap}
-            />
-        </LayerCard>
+        <>
+            {basemap.id === undefined ? (
+                <ComponentCover translucent>
```

Comment:
Remove `translucent`. The spinner looks better without it when it's no content to cover.